SU-SWS/stanford-jumpstart-deployer
# Upgrade VBO to 3.4 release

# READY FOR REVIEW

# Summary
- Update fixes the VBO issue for the manage/content debacle we had last week.
- 

# Needed By (Date)
- Sometime between now and the next sites release.

# Urgency
- Moderate

# Steps to Test

1. Build a JSA with this branch
2. Navigate to admin/mange
3. Select a number of items from the list of content
4. Select delete option from VBO options
5. Submit and confirm delete
6. Run cron
7. See correct nodes were deleted

# Affected Projects or Products
- All products and sites.

# Associated Issues and/or People
- https://www.drupal.org/project/views_bulk_operations/releases/7.x-3.4
